config/example/config.txt

```diff
@@ -426,6 +426,9 @@ RADIATION_LOWER_LIMIT 0.15
 ## Uncomment this to modify the number of actions permitted per interval before being kicked for spam.
 #MAX_ACTS_PER_INTERVAL 140
 
+## Is the panic bunker currently on?
+#PANIC_BUNKER

```

Comment:
Should be annotated as "on by default" or similar; it's a config setting, not a state var.